src/components/TodoList.tsx

```diff
@@ -0,0 +1,116 @@
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todos: Todo[]
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="todoapp__main">
```

Comment:
Wrap the entire list in a conditional rendering to hide the list and footer when there are no todos. You can achieve this by using a ternary operator or an '&&' operator.